### PR TITLE
[MMCA-4808] Amend copy of blue inset text box - remove the word 'the'

### PR DESCRIPTION
**Pull request by wg-hmrc**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -239,7 +239,7 @@ cf.authorities.request.received.link = Back to manage import duties and VAT acco
 cf.authorities.notification-panel.p1=The list of authorities you requested are available to download:
 cf.authorities.notification-panel.a.gb-authority=GB authorities (CSV)
 cf.authorities.notification-panel.a.xi-authority=Northern Ireland authorities (CSV)
-cf.authorities.notification-panel.p2=Your list was created on the {0}. It will be available to view for 10 days.
+cf.authorities.notification-panel.p2=Your list was created on {0}. It will be available to view for 10 days.
```
Comment (tyagikumarvineet) [inline conf/messages.en:242]: For the English message, existing is 'Your list was created on 25 November 2022'

But it should be 'Your list was created on 19th August 2022.' as per ticket. 
Could you please confirm with Sujay?

Comment (wg-hmrc) [reply conf/messages.en:242]: the date is dynamic @tyagikumarvineet

Comment (tyagikumarvineet) [reply conf/messages.en:242]: @wg-hmrc yes, could saw that earlier  as well that the date is dynamic but my query is whether it is correct cause as per ticket it should be 'Your list was created on 19th August 2022.' And it should be the same for both English and Welsh. In Welsh its static. Have you confirmed with Sujay?

Comment (tyagikumarvineet) [reply conf/messages.en:242]: @wg-hmrc I have discussed it with Sujay and date in Welsh translation should also be dynamic. He would raise a separate ticket to update the Welsh translation date to be dynamic. Hence I am resolving this

**Review by tyagikumarvineet — APPROVED** [review]

**Pull request merged**